app/core/env.ts

```diff
@@ -1,5 +1,5 @@
-import {app as electronApp, remote} from "electron"
-const app = electronApp || (remote && remote.app)
+import {app as electronApp} from "electron"
+const app = electronApp
```

Comment:
This can run from the renderer process, so we'll need a way to tell the renderer what the environment looks like. We can make an ipc call and set a global.